src/css/application.css

```diff
@@ -341,7 +341,7 @@ body {
 }
 
 .editors__label {
-  background-color: var(--color-low-contrast-gray);
+  background-color: #e2e2e2;
```

Comment:
Project style is to keep all CSS colors in a variable—you won’t find color literals elsewhere in CSS property definitions (I hope!).

Can we extract this into a variable?